package/Shaders/DeferredCompositeCS.hlsl

```diff
@@ -113,15 +120,16 @@ Texture3D<sh2> SkylightingProbeArray : register(t9);
 			specularIrradianceReflections = EnvReflectionsTexture.SampleLevel(LinearSampler, R, level).xyz;
 			specularIrradianceReflections = sRGB2Lin(specularIrradianceReflections);
 		}
-
-		color += reflectance * lerp(specularIrradiance, specularIrradianceReflections, skylightingSpecular);
+		finalIrradiance += lerp(specularIrradiance, specularIrradianceReflections, skylightingSpecular);
```

Comment:
Should integrate pbr directional ambient with skylighting occlusion.
I tried multiplying finalIrradiance with skylightingSpecular before adding the rest and it looks ok.